src/main/java/org/galatea/starter/ApiError.java

```diff
@@ -0,0 +1,24 @@
+package org.galatea.starter;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.http.HttpStatus;
+
+@Getter
```

Comment:
Don't need all of these annotations. [@Data](https://projectlombok.org/features/Data) is a convenience annotation which provides some of the others. Looking at how ApiError is used I suggest you need @Getter, @AllArgsConstructor and maybe @ToString.